pydynamo/test/sstable_test.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+from ..storage.memory.inmemorystore import InMemoryStore
+from ..storage.disk.sstable import SSTable
+from ..storage.error import StorageException
+from ..storage.disk.sstableiterator import SStableIterator
+from os import path
+
+
+class SStableTest(unittest.TestCase):
+    def setUp(self):
+        mem_store = InMemoryStore("test")
+        mem_store.set("mengying", "computerscience")
+        mem_store.set("kelly", "kpmg")
+        mem_store.set("chen", "facebook")
+        mem_store.set("jean", "deloitte")
+        self.sstable = SSTable("test", 0, mem_store, 0.1)
```

Comment:
you could create a temp folder for testing  https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp